src/NLog.Targets.GraylogHttp/GraylogMessageBuilder.cs

```diff
@@ -8,6 +8,10 @@ internal class GraylogMessageBuilder
         private const string NLogLogLevelPropertyName = "nlog_level_name";
         private readonly JsonObject _graylogMessage = new JsonObject();
         private static readonly DateTime _epochTime = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
+        private string _structuredLoggingParameterName;
+
+        public GraylogMessageBuilder(string structuredLoggingParameterName = null)

```

Comment:
Move this to a builder method. and out of the ctor

Something like WithStructuredLoggingParameter(string parameterName) {...)